src/libultra/io/cartrominit.c

```diff
@@ -1,3 +1,63 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/boot/cartrominit/osCartRomInit.s")
+OSPiHandle CartRomHandle;
+
+OSPiHandle* osCartRomInit(void) {
+    register u32 a;
+    register s32 status;
+    register u32 prevInt;
+    register u32 lastLatency;
+    register u32 lastPageSize;
+    register u32 lastRelDuration;
+    register u32 lastPulse;
+
+    static u32 D_800980D0 = 1;
```

Comment:
This is a boolean, I suggest something like `sCartRomNeedsInit` since if it's false it skips the initialization and just returns the handle right away. I tried to find the original name from the 2.0I dev cd but I was unsuccessful.

Reply:
That makes sense. Should it be s32 if it's a bool? Matches as either.

Reply:
iirc we like to keep bools s32 when we can? I'm not really sure.

Reply:
I'll make it s32 then.